src/universal-dapp-ui.js

```diff
@@ -53,6 +57,15 @@ UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address
   }
 
   instance.appendChild(title)
+  self.udapp.getBalanceInEther(address, (ballErr, balance) => {
+    $(instance.querySelector('div[class^="titleText"]')).innerHTML = `${contractName} at ${shortAddress} (${context}) (${balance} eth)`
+  })
+
+  setInterval(function () {
+    self.udapp.getBalanceInEther(address, (ballErr, balance) => {
```

Comment:
seems like there are some code duplication with https://github.com/ethereum/remix-ide/pull/1628/files#diff-b2d36c7664b525607de0fb789d7a689cR204

Reply:
i don't see why this is needed . seems like https://github.com/ethereum/remix-ide/pull/1628/files#diff-b2d36c7664b525607de0fb789d7a689cR709 should be enough for updating the ui with the new balance